blueocean-dashboard/src/main/js/components/RunDetailsTests.jsx

```diff
@@ -81,9 +78,9 @@ RunDetailsTests.propTypes = {
     params: PropTypes.object,
     isMultiBranch: PropTypes.bool,
     result: PropTypes.object,
-    testResults: PropTypes.object,
-    resetTestDetails: PropTypes.func,
-    fetchTestResults: PropTypes.func,
+    tests: PropTypes.object,
```

Comment:
Because there is an object in the store called test details and a React component called the same thing, all aliased and inconsistent throughout this area of code that left me trying to render "test results" array as a component and much time lost.

I suspect the naming convention wasn't good in the first place and that's why stuff was aliased everywhere.